src/minimization.h

```diff
@@ -0,0 +1,7 @@
+#include "layout.h"
+
+#ifndef LSVIS_MINIMIZATION_HPP
+#define LSVIS_MINIMIZATION_HPP
```

Comment:
Place includes *after* the header guard
```suggestion
#ifndef LSVIS_MINIMIZATION_HPP
#define LSVIS_MINIMIZATION_HPP

#include "layout.h"
```